extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-preview.js

```diff
@@ -0,0 +1,654 @@
+    'deviceHeight': 731,
+  },
+  {
+    'name': 'Pixel 3',
+    'width': 411,
+    'height': 686,
+    'deviceHeight': 823,
+  },
+  {
+    'name': 'iPhone 8 (Browser)',
+    'width': 375,
+    'height': 554,
+    'deviceHeight': 667,
+  },
+  {
+    'name': 'iPhone 8 (Native)',
+    'width': 375,
+    'height': 632,
+    'deviceHeight': 667,
+  },
+  {
+    'name': 'iPhone 11 (Browser)',
+    'width': 414,
+    'height': 724,
+    'deviceHeight': 896,
+  },
+  {
+    'name': 'iPhone 11 (Native)',
+    'width': 414,
+    'height': 795,
+    'deviceHeight': 896,
+  },
+  {
+    'name': 'iPhone 11 Pro (Browser)',
+    'width': 375,
+    'height': 635,
+    'deviceHeight': 812,
+  },
+  {
+    'name': 'iPhone 11 Pro (Native)',
+    'width': 375,
+    'height': 713,
+    'deviceHeight': 812,
+  },
+  {
+    'name': 'iPad (Browser)',
+    'width': 810,
+    'height': 1010,
+    'deviceHeight': 1080,
+  },
+  {
+    'name': 'OnePlus 5T',
+    'width': 455,
+    'height': 820,
+    'deviceHeight': 910,
+  },
+  {
+    'name': 'OnePlus 7 Pro',
+    'width': 412,
+    'height': 743,
+    'deviceHeight': 892,
+  },
+  {
+    'name': 'Desktop 1080p',
+    'width': 1920,
+    'height': 1080,
+    'deviceHeight': 1080,
+  },
+];
+
+/**
+ * Method that returns lowercase and remove non-alphanumeric, used on matching hashString.
+ * @param {string} name
+ * @return {string}
+ */
+function simplifyDeviceName(name) {
+  return name.toLowerCase().replace(/[^a-z0-9]/gi, '');
+}
+
+/**
+ * Get devices from the queryHash into list of devices
+ * @param {string} queryHash
+ * @return {any[]}
+ */
+function parseDevices(queryHash) {
+  const screenSizes = [];
+
+  queryHash.split(';').forEach((device) => {
+    const deviceSpecs = device.split(':');
+    let currSpecs = null;
+    if (deviceSpecs.length == 1) {
+      currSpecs = ALL_SCREEN_SIZES.find((el) => {
+        // Find first device that has prefix of the device name passed in.
+        const currDeviceName = simplifyDeviceName(el.name);
+        const specDeviceName = simplifyDeviceName(deviceSpecs[0]);
+        return (
+          currDeviceName.substring(0, specDeviceName.length) == specDeviceName
+        );
+      });
+    }
+    if (currSpecs) {
+      screenSizes.push(currSpecs);
+    }
+  });
+  return screenSizes;
+}
+
+/**
+ * @typedef {{
+ *  element: !Element,
+ *  player: !Element
+ *  chip: !Element,
+ *  width: number,
+ *  height: number,
+ *  deviceHeight: ?number,
+ *  custom: ?boolean,
+ * }}
+ */
+export let DeviceInfo;
+
+export class AmpStoryDevToolsTabPreview extends AMP.BaseElement {
+  /** @param {!Element} element */
+  constructor(element) {
+    super(element);
+
+    /** @private  {?string} */
+    this.storyUrl_ = null;
+
+    /** @private {!Array<DeviceInfo>} */
+    this.devices_ = [];
+  }
+
+  /** @override */
+  buildCallback() {
+    const styleTag = this.element.ownerDocument.createElement('style');
+    styleTag.textContent = CSS;
+    this.element.appendChild(styleTag);
```

Comment:
Please use the default CSS, cf my comment on `bundles.config.js`. This is all unneeded